enforcer-rules/README.md

```diff
@@ -0,0 +1,89 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+any [linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Class path and dependencySection flag
+
+The value of the dependencySection element determines whether the rule checks the dependencies in
+the `dependencies` section or the `dependencyManagement` section.
+
+- When `DEPENDENCIES` (default value), the rule checks the class path calculated from the project's
+  `dependencies` section and their transitive dependencies.
+- When `DEPENDENCY_MANAGEMENT`, the rule checks a class path consisting of artifacts in the
```

Comment:
"When the value of the dependencySection element is DEPENDENCY_MANAGEMENT"

Also, is it customary for these values to be upper case?

Reply:
Updated to more simpler form.

>upper case?

Upper case indicates it's enum. Other rules use it. For example, enforcer-rules use "WARN" or "ERROR" for level element.
https://maven.apache.org/enforcer/maven-enforcer-plugin/usage.html